client/src/pages/messaging/Style.module.css

```diff
@@ -1,8 +1,7 @@
 .messageContainer {
   padding-bottom: 50px;
   /* height fallback */
-  height: 90vh;
-  height: calc(100% - 50px);
+  height: calc(100vh - 50px);
```

Comment:
`vh` doesn't work in mobile as expected (due to address bar behavior). Use % instead.

Reply:
Using % is making the message window like this
![Screenshot from 2020-08-12 09-38-46](https://user-images.githubusercontent.com/62807661/89974095-be4e9600-dc7f-11ea-855a-1cbe409a9360.png)

Reply:
Okay, this is resolved. Added %.